backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/tax_accounting_v3/report.jrxml

```diff
@@ -29,12 +29,13 @@
 		<defaultValueExpression><![CDATA[]]></defaultValueExpression>
 	</parameter>
 	<query language="sql"><![CDATA[SELECT *
-FROM de_metas_acct.taxaccounts_report($P{AD_Org_ID},
+FROM de_metas_acct.report_taxaccounts($P{AD_Org_ID},
                                       $P{account_id},
                                       $P{C_VAT_Code_ID},
                                      $P{DateFrom}, 
 									$P{DateTo},
                                    $P{showdetails},
+									null,
```

Comment:
if u would use named parametes, IMHO you don't need this null here.
p_Level shall have a default value of `null`